labelbox/data/annotation_types/data/tiled_image.py

```diff
@@ -0,0 +1,364 @@
+import math
+import logging
+from enum import Enum
+from typing import Optional, List, Tuple
+from concurrent.futures import ThreadPoolExecutor
+
+import requests
+import numpy as np
+import tensorflow as tf
+from retry import retry
+from pyproj import Transformer
+from pydantic import BaseModel, validator, conlist
+from pydantic.class_validators import root_validator
+
+from ..geometry import Point
+from .base_data import BaseData
+from .raster import RasterData
+"""TODO: consider how to swap lat,lng to lng,lt when version = 2...
+should the bounds validator be inside the TiledImageData class then, 
+since we need to check on Version?
+"""
+
+VALID_LAT_RANGE = range(-90, 90)
+VALID_LNG_RANGE = range(-180, 180)
+TMS_TILE_SIZE = 256
+MAX_TILES = 300  #TODO: thinking of how to choose an appropriate max tiles number. 18 seems too small, but over 1000 likely seems too large
+TILE_DOWNLOAD_CONCURRENCY = 4
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+
+class EPSG(Enum):
+    """ Provides the EPSG for tiled image assets that are currently supported.
+    
+    SIMPLEPIXEL is Simple that can be used to obtain the pixel space coordinates
+
+    >>> epsg = EPSG()
+    """
+    SIMPLEPIXEL = 1
```

Comment:
Aren't we also going to have simple and simple pixel. Simple is zoom 0 and is an actual projection. Simple pixel is more like 'no projection'.

Reply:
Maybe we can change this to `EPSG.None` to indicate that the coordinates do not have an epsg.

Reply:
I thought we decided to only keep SIMPLEPIXEL and not both Simple, and SimplePixel 

Wouldn't Simple at zoom 0 also be the same as simple pixel if there is no projection? 

Notes:

===========================
- Use cases we need to support
4326 to SIMPLEPIXEL
SIMPLESIMPLEPIXEL to 4326
4326 to 3857
===========================